app/src/main/res/layout/activity_main.xml

```diff
@@ -2,32 +2,39 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/activity_main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".activity.MainActivity">
 
     <TextView
         android:id="@+id/visor_operation"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/visor_operation"
-        android:textAlignment="gravity"
+        android:layout_height="@dimen/visor_operation_dim"
+        android:layout_marginTop="@dimen/marginOperation"
+        android:layout_marginLeft="@dimen/marginOperation"
+        android:textSize="@dimen/textOperationSize"
         app:layout_constraintBottom_toTopOf="@id/visor_result"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="SpUsage"
```

Comment:
Please, modify this why are you being recommended to use SP style to format a textSize

Reply:
done